src/main/java/dk/kb/kaltura/client/DsKalturaClient.java

```diff
@@ -82,6 +83,24 @@ public DsKalturaClient(String kalturaUrl, String userId, int partnerId, String a
     }
 
 
+    /** 
+     * <p>
+     * Delete a stream and all meta-data for the record in Kaltura.
+     * It can not be restored in Kaltura and must be uploaded again if deleted by mistake.       
+     * </p>
+     *  
+     * @param entryId The unique id in the Kaltura platform for the stream
+     * @return True if record was found and deleted. False if the record with the entryId could not be found in Kaltura.
+     * @throws IOException if Kaltura API called failed.
+     */
+    public boolean deleteStreamByEntryId(String entryId) throws IOException{                
+         Client clientSession = getClientInstance();        
+         DeleteMediaBuilder request = MediaService.delete(entryId);         
+         Response<?> execute = APIOkRequestsExecutor.getExecutor().execute(request.build(clientSession)); // no object in response. Only status
```

Comment:
I am so glad we are doing this client around the kaltura API, because this is so weird to look at.